app/lib/widgets/gallery/gallery.dart

```diff
@@ -19,6 +20,7 @@ class FlexGallery extends StatefulWidget {
   final List<String> imageUrls;
   final List<String>? thumbnailUrls;
   final double borderRadius;
+  final double thumbRowPaddingX;
```

Comment:
rowpaddingx seems very specific.. I wonder if there's a better name.

its verbose.. but maybe consider 
`horizontalThumbnailPadding` or
`horizontalItemPadding`

or maybe instead of a double, we can define the edge insets on the widget?
`final EdgeInsets galleryItemInsets`..